doc/xml/daps_user_output.xml

```diff
@@ -235,4 +235,40 @@
    
   </variablelist>
  </sect1>
+ <sect1 id="sec.daps.user.output.xmlformat">
+  <title>Pretty XML Formatting</title>
+  <para>
+	  Sometimes it might be usefull to have propper formatted XML sources.
```

Comment:
simplify: "Sometimes it might be" -> "It can be" (takes out some of the indirection here)
typos: "usefull" -> "useful"
typo: "propper" -> "properly" (need to use an adverb here since we're describing an adjective)